src/containers/Employees/index.js

```diff
@@ -37,11 +38,12 @@ export default class Employees extends React.Component {
     this.setState({ selectedMojos: [] });
   };
 
-  sendNotification = slackID => {
-    console.log(slackID);
+  sendNotification = selectedMojos => {
+    const channelName = Config.CHANNEL;
     const { navigation } = this.props;
     const guest = navigation.getParam('guestName');
-    console.log(guest);
+    sendMessageToChannel(channelName, guest, selectedMojos);
```

Comment:
This is okay as is, but `await sendMessageToChannel` would allow us to redirect to the welcome screen _after_ the message finished sending. Since we don't have a loading indicator/error handling at the moment, no change necessary.